app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,46 @@
+class QuestionsController < ApplicationController
+
+  before_action :found_test, only: [:show, :new, :create]
+
+  rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_question_not_found
+
+  def index
+    @test = Test.find(params[:test_id])
+    @questions = @test.questions.all
+    render inline: '<%= @questions.pluck(:id, :body).join("\n")%>'
+  end
+
+  def new
+  end
+
+  def show
+    # render inline: '<%= Question.find(params[:id]).body%>'
+    # render json: Question.find(params[:id])
+    @question = @test.questions.find(params[:id])
+    render inline: '<%= @question.body%>'
+  end
+
+  def create
+    @question = @test.questions.create(params.require(:question).permit(:body))
+    render inline: '<p>Результат создания вопроса: <%=  !!@question.valid? %></p>'
+    # render plain: question.inspect
+    # redirect_to test_questions_path
```

Comment:
```suggestion
    @question = @test.questions.build(question_params)
    if @question.save
      redirect_to @question
    else
      render :new
    end
```
.build для создания объекта, а потом метод .save, что позволяет отделить создание объекта от сохранения и лучше контролировать процесс, плюс проверка на успешное сохранение вопроса, и в зависимости от этого либо происходит редирект, либо выводится форма создания вопроса заново